mirascope/core/_internal/utils.py

```diff
@@ -110,6 +121,7 @@ def convert_function_to_base_tool(
     Args:
         fn: The function to convert.
         base: The `BaseToolT` type to which the function is converted.
+        __doc__: The docstring to use for the constructed `BaseToolT` type.
```

Comment:
should this just be `doc` instead of `__doc__` since it's an input argument?

Reply:
I thought the same thing at first, but then I thought that __doc__ would convey the intention better, so I decided on this one.